pkg/pillar/scripts/device-steps.sh

```diff
@@ -295,8 +295,8 @@ access_usb() {
     if [ -n "$SPECIAL" ] && [ -b "$SPECIAL" ]; then
         echo "$(date -Ins -u) Found USB with DevicePortConfig: $SPECIAL"
         if ! mount -t vfat "$SPECIAL" /mnt; then
-            # XXX !? will be zero
-            echo "$(date -Ins -u) mount $SPECIAL failed: $?"
+            ret_code=$?
```

Comment:
Cannot understand code from what command we expect here.

Reply:
Intent is to get exit code from the mount. Perhaps that doesn't work in the context of the if statement? (I don't have an easy way to test this - but it isn't worse than the old code getting the exit code from the data command.)

Reply:
Probably it is better to move the command from the if and check for $ret_code -ne 0 inside if?

Reply:
Fixed